google-cloud-bigquery/lib/google/cloud/bigquery/copy_job.rb

```diff
@@ -130,6 +130,19 @@ def write_empty?
           disp == "WRITE_EMPTY"
         end
 
+        ##
```

Comment:
Does these files need to add a require for the encryption class? I understand it is likely required somewhere else already, but do we want a require to document their use in these files?